src/test/java/salesforce/scenarios/entity/EntitiesSteps.java

```diff
@@ -67,29 +68,29 @@ public void iSetThePathParamsAndUpdatedBodyToRequest(final DataTable jsonData) t
     public void iSetTheEndpointAndSendTheRequestWithBody(final String endpoint) throws JsonProcessingException {
         logger.info("=================== Create When ===========================");
         logger.info(body);
-        apiResponse = ApiRequestManager.create(endpoint, pathParams, body);
+        apiResponse = ApiRequestManager.execute(endpoint, pathParams, body, ApiMethod.POST);
```

Comment:
Maybe it is possible to set the request method trough the features and reduce the steps with it.

Reply:
Good suggestion. We will discuss it for next PR